### PR TITLE
Feature/payment methods

### PR DESCRIPTION
**Pull request by MaxAstin**
photo_2023-04-18 00 32 43

### CODE REVIEW

File: shared/src/commonMain/kotlin/com/bunbeauty/shared/domain/feature/payment/GetPaymentMethodListUseCase.kt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package com.bunbeauty.shared.domain.feature.payment
+
+import com.bunbeauty.shared.domain.model.payment_method.PaymentMethod
+import com.bunbeauty.shared.domain.repo.PaymentRepo
+
+class GetPaymentMethodListUseCase(
```
Comment (xidealo) [inline shared/src/commonMain/kotlin/com/bunbeauty/shared/domain/feature/payment/GetPaymentMethodListUseCase.kt:6]: тесты

File: app/src/main/java/com/bunbeauty/papakarlo/feature/profile/screen/profile/ProfileFragment.kt [hunk 2]
```diff
@@ -216,52 +222,33 @@ class ProfileFragment : BaseFragmentWithSharedViewModel(R.layout.fragment_profil
...
-
             NavigationIconCard(
                 modifier = Modifier.fillMaxWidth(),
                 iconId = R.drawable.ic_settings,
                 iconDescription = R.string.description_ic_settings,
                 labelStringId = R.string.action_profile_settings,
                 onClick = onSettingsClick
             )
-
+            Spacer(modifier = Modifier.height(8.dp))
```
[the remark this answers is not in the document]
Comment (MaxAstin) [reply app/src/main/java/com/bunbeauty/papakarlo/feature/profile/screen/profile/ProfileFragment.kt]: fixed

File: shared/src/commonMain/kotlin/com/bunbeauty/shared/domain/feature/payment/GetPaymentMethodListUseCase.kt (hunk 1 above)
Comment (MaxAstin) [reply shared/src/commonMain/kotlin/com/bunbeauty/shared/domain/feature/payment/GetPaymentMethodListUseCase.kt:6]: fixed

File: shared/src/commonMain/kotlin/com/bunbeauty/shared/data/network/api_result_handler/ApiResultHandler.kt [hunk 3]
```diff
@@ -0,0 +1,19 @@
+package com.bunbeauty.shared.data.network.api_result_handler
+
+import com.bunbeauty.shared.data.network.ApiError
+import com.bunbeauty.shared.data.network.ApiResult
+import com.bunbeauty.shared.data.network.model.ListServer
+
+interface ApiResultHandler {
```
[the remark this answers is not in the document]
Comment (MaxAstin) [reply shared/src/commonMain/kotlin/com/bunbeauty/shared/data/network/api_result_handler/ApiResultHandler.kt]: fixed

**Pull request merged**